WordPress/Classes/ViewRelated/Media/MediaLibraryViewController.swift

```diff
@@ -132,11 +134,17 @@ class MediaLibraryViewController: WPMediaPickerViewController {
 
             if blog.userCanUploadMedia && assetCount > 0 {
                 let addButton = UIBarButtonItem(barButtonSystemItem: .add, target: self, action: #selector(addTapped))
+                addButton.accessibilityLabel = NSLocalizedString("Add", comment: "Accessibility label for add button in nav bar")
+                addButton.accessibilityHint = NSLocalizedString("Add new media", comment: "Accessibility hint for add button in nav bar")
+
                 barButtonItems.append(addButton)
             }
 
             if blog.supports(.mediaDeletion) && assetCount > 0 {
                 let editButton = UIBarButtonItem(barButtonSystemItem: .edit, target: self, action: #selector(editTapped))
+                editButton.accessibilityLabel = NSLocalizedString("Edit", comment: "Accessibility label for edit button in nav bar")
```

Comment:
The comment here could mention `edit button to enable multi selection mode in the user's media library` or something similar.

Reply:
No prob - done